build/websocket-scenarios.yml

```
@@ -16,8 +16,14 @@ parameters:
   type: object
   default: 
 
-  - displayName: Echo
-    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --property scenario=echo
+  - displayName: Echo-10000
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000--property scenario=echo
+  - displayName: Echo-100
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=100 --property scenario=echo
+  - displayName: Echo-Compression-100
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=100 --variable compress=true --property scenario=echo --application.buildArguments \"/p:UseCompression=true\"
+  - displayName: Echo-Compression-10000
+    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000 --variable compress=true --property scenario=echo --application.buildArguments \"/p:UseCompression=true\"
```

```suggestion
    arguments: --scenario websocket $(websocketJobs) --variable scenario=echo --variable size=10000 --variable compress=true --property scenario=echo --application.buildArguments \"/p:UseCompression=true\" --property size=10000 --property compress=true
```

You can also create other collections (100, 1000, 1000) and (true/false) to do embedded loops to generate all command lines. Check the containers benchmarks file for instance.

---

Changed to use loops, please take a look :)